Assignees.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+namespace ST;
+
+use ParserOutput;
+use SMW\ApplicationFactory;
+use SMW\DIWikiPage;
+use SMWDataItem;
+use User;
+use WikiPage;
+
+/** @todo: rename TaskDiff something similar */
+class Assignees {
```

Comment:
Why did you copy this class?

Reply:
hello Jeroen, in the root folder ? it was an error of github file upload, you can delete it